File: mahaam-api-java/src/main/java/mahaam/infra/monitor/MonitorModel.java
package mahaam.infra.monitor;

import java.util.UUID;

public class MonitorModel {
	public static class Health {
		public UUID id;
		public String apiName;
		public String apiVersion;
		public String nodeIP;
		public String nodeName;
		public String envName;
	}

	public static class Traffic {
		public UUID id;
		public UUID healthId;
		public String method;
		public String path;
		public Integer code;
		public Long elapsed;
		public String headers;
		public String request;
		public String response;

		public Traffic() {
		}

		public Traffic(
				UUID id,
				UUID healthId,
				String method,
				String path,
				Integer code,
				Long elapsed,
				String headers,
				String request,
				String response) {
			this.id = id;
			this.healthId = healthId;
			this.method = method;
			this.path = path;
			this.code = code;
			this.elapsed = elapsed;
			this.headers = headers;
			this.request = request;
			this.response = response;
		}
	}

	public static class TrafficHeaders {
		public UUID userId;
		public UUID deviceId;
		public String appVersion;
		public String appStore;
	}
}

File: mahaam-api-ts/src/infra/monitor/monitor.model.ts
export interface Traffic {
  id: string;
  healthId: string;
  method: string;
  path: string;
  code?: number | null;
  elapsed?: number | null;
  headers?: string | null;
  request?: string | null;
  response?: string | null;
}

export interface TrafficHeaders {
  userId?: string | null;
  deviceId?: string | null;
  appVersion?: string | null;
  appStore?: string | null;
}

export interface Health {
  id: string;
  apiName?: string | null;
  apiVersion?: string | null;
  nodeIP?: string | null;
  nodeName?: string | null;
  envName?: string | null;
}

export interface HealthInfo {
  app: string;
  version: string;
  env: string;
}
